packages/react/src/components/Masthead/README.md

```diff
@@ -37,40 +37,42 @@ MASTHEAD_L1=true
 > [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/.env.example)
 > for more information
 
-## Navigation Types
+## navigation (String | Object | null)
 
-| Name      | Description                          |
-| --------- | ------------------------------------ |
-| `default` | Default navigation data from IBM.com |
-| `custom`  | Custom navigation data               |
-| `none`    | No navigation                        |
+| Behavior           | Data Type | Description                                 | Example                             |
+| ------------------ | --------- | ------------------------------------------- | ----------------------------------- |
+| default navigation | String    | Default navigation data from IBM.com        | `<Masthead navigation="default" />` |
+| custom navigation  | Object    | Pass in custom navigation data as an object | `<Masthead navigation={myNavObj}/>` |  |
+| none               | null      | No navigation                               | `<Masthead />`                      |
 
 > 💡 `Custom` navigation data must follow the same structure and key names as
 > `default`. See
 > [this](https://www.ibm.com/common/v18/js/data/jsononly/usen.json) for an
 > example.
 
-## Options
+## platform (Object)
```

Comment:
May be a bit silly, but shouldn't we use backticks like `navigation` and `platform`?